api/actors.yml

```diff
@@ -0,0 +1,149 @@
+      info: 
+          oneOf: 
+            - $ref: '#/models/AccountActorState'
+            - $ref: '#/models/InitActorState'
+            - $ref: '#/models/MinerActorState'
+            - $ref: '#/models/PaymentBrokerActorState'
+            - $ref: '#/models/StorageMarketActorState'
+  AccountActorState:
+    $ref: '#/models/EmptyActorState'
+  EmptyActorState:
+    type: object 
+    description: There is no additional state exposed by this actor
+  InitActorState:
+    type: object
+    properties:
+      nextId: 
+        type: number
+        description: The next numerical ID to allocate to a newly instantiated Actor
+  MinerActorState:
+    type: object
+    properties: 
+      owner:
+        type: string
+        description: The address of the account that owns this miner. Income and returned are paid to this address. This address is also allowed to change the worker address for the miner.
+      worker:
+        type: string
+        description: The address of the worker account for this miner. This will be the key that is used to sign blocks created by this miner, and sign messages sent on behalf of this miner to commit sectors, submit PoSts, and other day to day miner activities.
+      peerId: 
+        type: string
+        description: PeerID references the libp2p identity that the miner is operating.
+      sectorSize:
+        type: number
+        description: SectorSize is the amount of space in each sector committed to the network by this miner.
+      dePledgedCollateral:
+        type: number
+        description: Collateral that is waiting to be withdrawn
+      dePledgeTime: 
+        type: number
+        description: Time at which the depledged collateral may be withdrawn
+      sectors: 
+        type: array
+        items: 
+          $ref: 'actors.yml#/models/Sector'
+      provingSet:
+        type: array
+        items: 
+          $ref: 'actors.yml#/models/Sector'
+      currentFaultSet:
+        type: string
+      nextFaultSet:
+        type: string
+      nextDoneSet:
+        type: string
+      arbitratedDeals:
+        type: string
+      power:
+        type: number
+      slashedSet:
+        type: array
+        items: 
+          $ref: 'actors.yml#/models/Sector'
+      slashedAt:
+        type: number
+      ownedStorageCollateral:
+        type: number
+      provingPeriodEnd:
+        type: number
+  PaymentBrokerActorState:
+    $ref: '#/models/EmptyActorState'
+  Sector:
+    type: object
+    properties:
+      id:
+        type: string
+      commR:
+        type: string
+  StorageMarketActorState:
+    type: object
+    properties:
+      totalStorage: 
+        type: number
+        description: Filecoin's total committed storage as of the current block.
+endpoints:
+  actors_collection:
+    get:
+      operationId: listActors
+      summary: List Actors
+      description: List Actors currently known to this node in ID order
+      tags:
+        - Actors
+      parameters:
```

Comment:
This, and all state query operations, must take a tipset key as an optional parameter, indicating the point in the chain history at which to query. Whether or not it's specified, the result should include an envelope that specifies the tipset at which the query was actually evaluated.